### PR TITLE
feat(slo-stat-bot): Added scheduler for cron events

### PR DESCRIPTION
**Pull request by hershey127**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/repo-automation-bots/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: packages/slo-stat-bot/src/slo-bot.ts [hunk 1]
```diff
@@ -40,17 +63,16 @@ async function handleIssues(
   owner: string,
   repo: string,
   type: string,
+  number: number,
```
Comment (JustinBeckwith) [inline packages/slo-stat-bot/src/slo-bot.ts:66]: This is probably too many parameters for a function to be easily readable.  It feels like many of these fields would be available on an `Issue` interface.  I'm also guessing in the `Octokit` namespace there would be a `Issue` type we could readily use here.

File: packages/slo-stat-bot/src/slo-bot.ts [hunk 2]
```diff
@@ -99,6 +121,34 @@ async function getSloFile(
...
+  try {
+    const state = 'open';
+    const issueList = await github.issues.listForRepo({
+      owner,
+      repo,
+      state,
+    });
+    return issueList.data;
+  } catch (err) {
+    console.error(
```
Comment (JustinBeckwith) [inline packages/slo-stat-bot/src/slo-bot.ts:145]: A few things here!
- For all logging, we're now using `logger.error`, where `logger` comes from `import {logger} from 'gcf-utils';`
- When dealing with errors, it's import you both log the error, *and* it's call stack. The pattern we're starting to widely use here is:

```js
err.message = `Error in getting list of issues from repo ${repo}: ${err.message}`;
logger.error(err);
```

Logging the entire error object ensures all fields will be included and we use the right logging level in Cloud Logging.  Please do a double check and make sure this is happening broadly!

File: packages/slo-stat-bot/src/slo-bot.ts (hunk 1 above)
Comment (orthros) [reply packages/slo-stat-bot/src/slo-bot.ts:66]: I agree; I suggested she do a second PR to do that fix in order to keep this PR relatively small and stable. Let us know if you want to add this to this PR though

File: packages/slo-stat-bot/src/slo-bot.ts (hunk 2 above)
Comment (hershey127) [reply packages/slo-stat-bot/src/slo-bot.ts:145]: Sorry that's my bad. Didn't catch that inconsistency but will fix it!

File: packages/slo-stat-bot/src/slo-bot.ts (hunk 1 above)
Comment (hershey127) [reply packages/slo-stat-bot/src/slo-bot.ts:67]: As Colin mentioned I just created the new PR which includes the issue types: https://github.com/googleapis/repo-automation-bots/pull/795

**Pull request closed without merging**